diesel_derives/src/identifiable.rs

```diff
@@ -14,12 +14,14 @@ pub fn derive(item: syn::DeriveInput) -> Result<quote::Tokens, Diagnostic> {
     ref_generics.params.push(parse_quote!('ident));
     let (ref_generics, ..) = ref_generics.split_for_impl();
 
-    let (field_ty, field_access): (Vec<_>, Vec<_>) = model
+    let (ref_field_ty, ref_field_access): (Vec<_>, Vec<_>) = model
         .primary_key_names
         .iter()
         .filter_map(|&pk| model.find_column(pk).emit_error())
         .map(|f| (&f.ty, f.name.access()))
         .unzip();
+    let direct_field_ty = ref_field_ty.clone();
```

Comment:
Can we not use the same variable twice in `quote!`?

Reply:
@sgrif I personally didn't find a way to do it when I needed it

Reply:
So it seems it is possible to use the same variable in `quote!` twice.
Internally `quote!` seems to use `into_iter` so passing a `Vec<_>` to quote will cause an error about a moved variable, but if we use a reference instead it works :wink: